tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/api/rest/v1/controllers/FormController.java

```diff
@@ -46,22 +51,36 @@ public class FormController extends ApiErrorController {
             description =
                 "An error is returned when the form with the `formId` and `processDefinitionKey` is not found.",
             responseCode = "404",
+            content =
+                @Content(
+                    mediaType = MediaType.APPLICATION_PROBLEM_JSON_VALUE,
+                    schema = @Schema(implementation = Error.class))),
+        @ApiResponse(
+            description =
+                "A forbidden error is returned when user does not have permission read user tasks.",
+            responseCode = "403",
             content =
                 @Content(
                     mediaType = MediaType.APPLICATION_PROBLEM_JSON_VALUE,
                     schema = @Schema(implementation = Error.class)))
       })
   @GetMapping("{formId}")
   public ResponseEntity<FormResponse> getForm(
-      @PathVariable @Parameter(description = "The ID of the form.", required = true) String formId,
+      @PathVariable @Parameter(description = "The ID of the form.", required = true)
+          final String formId,
       @RequestParam
           @Parameter(description = "Reference to the process definition.", required = true)
-          String processDefinitionKey,
+          final String processDefinitionKey,
       @RequestParam(required = false)
           @Parameter(
               description = "The version of the form. Valid only for deployed forms.",
               required = false)
-          Long version) {
+          final Long version) {
+    if (!tasklistPermissionServices.hasPermissionToReadProcessDefinition(WILDCARD_RESOURCE)) {
```

Comment:
In the V2 API, we are checking `ReadUserTask` on `ProcessDefinition` in https://docs.camunda.io/docs/next/apis-tools/orchestration-cluster-api-rest/specifications/get-user-task-form/. For consistency (avoiding unexpected behavior for users that a resource may be exposed with different permissions at different endpoints), I suggest we do it here in the same way.